test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -18,6 +18,8 @@
 @pytest.mark.parametrize("ec2_instance_type", SMDEBUG_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
     # TODO: Remove this once test timeout has been debugged (failures especially on p2.8xlarge)
+    if is_tf2(training) and "2.3.0" in training and SMDEBUG_EC2_GPU_INSTANCE_TYPE == "p2.8xlarge":
```

Comment:
This line might not be correct. Should be `if "p2.8xlarge" in SMDEBUG_EC2_GPU_INSTANCE_TYPE:`

Reply:
`get_ec2_instance_type()` returns a list

Reply:
+1

Reply:
good catch zhuo